### PR TITLE
Add fixer for OCaml ocp-indent

### PR DESCRIPTION
**Pull request by mkanenobu**
Add docs and test

A document error has occurred but it has also occurred in f0f0cc3c186c46f437d62e6b3e01a45d2a3aaf47

### CODE REVIEW

File: autoload/ale/fixers/ocp_indent.vim [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+call ale#Set('ocaml_ocp_indent_config', '')
+
+function! ale#fixers#ocp_indent#Fix(buffer) abort
+    let l:executable = ale#Var(a:buffer, 'ocaml_ocp_indent_executable')
+    let l:config = ale#Var(a:buffer, 'ocaml_ocp_indent_config')
+    let l:options = ale#Var(a:buffer, 'ocaml_ocp_indent_options')
+
+    return {
+    \   'command': ale#Escape(l:executable)
+    \       . (empty(l:config) ? '' : ' --config="' . l:config . '"')
```
Comment (w0rp) [inline autoload/ale/fixers/ocp_indent.vim]:
```suggestion
    \       . (empty(l:config) ? '' : ' --config=' . ale#Escape(l:config))
```

The command line argument here needs to be escaped like so. Escaping works differently on Unix and Windows. Update the tests accordingly.

Comment (mkanenobu) [reply autoload/ale/fixers/ocp_indent.vim]: Thank you for review!

I'll fix it.  
But I don't delete double quotes because ocp-indent needs them.

**Comment by mkanenobu** [comment]
Updated.  
Please check.

File: autoload/ale/fixers/ocp_indent.vim [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+function! ale#fixers#ocp_indent#Fix(buffer) abort
+    let l:executable = ale#Var(a:buffer, 'ocaml_ocp_indent_executable')
+    let l:config = ale#Var(a:buffer, 'ocaml_ocp_indent_config')
+    let l:options = ale#Var(a:buffer, 'ocaml_ocp_indent_options')
+
+    return {
+    \   'command': ale#Escape(l:executable)
+    \       . (empty(l:config) ? '' : ' --config=' . ale#Escape(l:config))
+    \       . (empty(l:options) ? '': ' ' . ale#Escape(l:options))
```
Comment (w0rp) [inline autoload/ale/fixers/ocp_indent.vim]: Don't escape `l:options`, as that's a string representing many options.

```suggestion
    \       . (empty(l:options) ? '': ' ' . l:options)
```

Comment (mkanenobu) [reply autoload/ale/fixers/ocp_indent.vim]: Fix complete.

**Comment by mkanenobu** [comment]
Updated.

**Comment by w0rp** [comment]
Cheers! :beers:

**Comment by mkanenobu** [comment]
Thank you!

**Pull request merged**